packages/devtools_app/lib/src/screens/vm_developer/vm_class_display.dart

```diff
@@ -6,9 +6,14 @@ import 'package:flutter/material.dart';
 import 'package:vm_service/vm_service.dart';
 
 import '../../primitives/utils.dart';
+import '../../shared/common_widgets.dart';
+import '../../shared/table.dart';
+import '../../shared/theme.dart';
 import 'vm_developer_common_widgets.dart';
 import 'vm_object_model.dart';
 
+const displayClassInstances = false;
```

Comment:
Can you add a TODO explaining that this functionality needs to be implemented?

Reply:
Sure